matlab/scripts/populate_sync.m

```diff
@@ -0,0 +1,7 @@
+while 1
+	try
+        parpopulate(stimulus.Sync, experiment.AutoProcessing)
+        pause(1000);
```

Comment:
rather than `try ... catch`, get a return argument from `parpopulte`:
```matlab
k = parpopulate(stimulus.Sync, experiment.AutoProcessing)
```
This has the same effect as `suppress_errors` in datajoint-python.

Reply:
oh, ok, thanks